src/style.css

```
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+}
+
+.search-form {
+  margin: 71px auto 0 48px;
+  width: 900px;
```

```suggestion
  width: 100%;
```
do not hardcode sizes of self-stretched elements